src/views/user/Login.vue

```diff
@@ -83,7 +84,7 @@ export default {
     login () {
       authService.login(this.form, 'error')
```

Comment:
Delete the second parameter, `'error'`. It was just used for testing.
Search in the whole project and if it is called with two arguments just delete the second one.